.github/workflows/node-integrate.yml

```diff
@@ -134,29 +136,30 @@ jobs:
       - name: Install main project dependencies
         if: steps.cacheNodeModules.outputs.cache-hit != 'true'
         run: |
+          cd node
           npm update --save-dev --no-save
       - name: Install packages/aws-lambda-otel-extension dependencies
         if: steps.cacheAwsLambdaOtelExtensionNodeModules.outputs.cache-hit != 'true'
         run: |
-          cd packages/aws-lambda-otel-extension
+          cd node/packages/aws-lambda-otel-extension
           npm update --no-save
           npm update --save-dev --no-save
       - name: Install packages/aws-lambda-otel-extension/opt/otel-extension dependencies
         if: steps.cacheAwsLambdaOtelExtensionOptOtelExtensionNodeModules.outputs.cache-hit != 'true'
         run: |
-          cd packages/aws-lambda-otel-extension/opt/otel-extension
+          cd node/packages/aws-lambda-otel-extension/opt/otel-extension
           npm update --no-save
       - name: Install packages/aws-lambda-otel-extension/test/fixtures/lambdas dependencies
```

Comment:
This workflow file is strictly for Node.js (so stuff we maintain in `node` folder), so to avoid extra noise I think it's fine to leave it as that.

Idea is to have other runtimes backed by different workflow files